src/main/java/duke/Task.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+/**
+ * The Task Class is the parent class to ToDo, Deadline and Event Classes
+ * @author qiujingying
+ * @version 1.0
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    private TaskType type = TaskType.TASK;
+    protected String time;
+
+    /**
+     * Creates a Task object with a description and time
```

Comment:
According to checkstyle, the first sentence in a Javadoc (aka the summary/description of the class/method) should end with a period '.'

This applies not just for this line of Javadoc but for all the other Javadocs!